modules/creating-a-custom-training-image.adoc

```diff
@@ -8,9 +8,59 @@ You can create a custom training image by adding packages to a base training ima
 .Prerequisites
 
 * You can access the training image that you have chosen to use as the base for your custom image. 
++
+Select the image based on the _image type_ (for example, Ray or KFTO), the _accelerator framework_ (for example, CUDA for NVIDIA GPUs, or ROCm for AMD GPUs), and the _Python version_ (for example, 3.9 or 3.11).
++
+*Note:* The Python version in the training image must be the same as the Python version in the {productname-short} workbench where you plan to use the image.
```

Comment:
That constraint is specific to Ray and does not apply to the Kubeflow training operator.

Reply:
Removed from this section, and added to the _Running ... from pipelines_ section. (Already included in _Running ...from notebooks_ section.)